sdl2promela/sdl/model.py

```diff
@@ -1451,6 +1451,54 @@ def convert(source: ogAST.Decision) -> Action:
     return None
 
 
+def _parseObserverInput(desc: str) -> Tuple[ObserverAttachmentInfo, str]:
+    info = ObserverAttachmentInfo()
+    parameter = ""
+    if desc.lower().startswith("input"):
+        info.kind = ObservedSignalKind.INPUT
+    elif desc.lower().startswith("output"):
+        info.kind = ObservedSignalKind.OUTPUT
+    else:
+        raise ValueError("Cannot parse continuous signal")
```

Comment:
Just to make sure - are we still supporting continuous signals on model variables, e.g., st.myfunct.myvar >= 5?

Reply:
OK, this is ensured by filtering in __gather_continuous_signals, is that right?

Reply:
Yes, this is still supported, with tests.